### PR TITLE
46 funds and reorder fields

### PR DESCRIPTION
**Pull request by shaneallgeier**
This PR swaps Type & Subtype, adds the funds field, renames "goal" to "goals", and adds some helpful text to the conclusion field.

### CODE REVIEW

File: exdb/models.py [hunk 1]
```diff
@@ -59,25 +59,36 @@ class Experience(models.Model):
...
     attendance = models.IntegerField(null=True, blank=True)
     created_datetime = models.DateTimeField(default=now, blank=True)
-    recognition = models.ManyToManyField(Section, blank=True)
     status = models.CharField(max_length=2, choices=STATUS_TYPES, default=STATUS_TYPES[1][0])
     next_approver = models.ForeignKey(settings.AUTH_USER_MODEL, blank=True, null=True, related_name='approval_queue')
-    conclusion = models.TextField(blank=True)
+    funds = models.CharField(max_length=2, choices=FUND_TYPES, blank=True)
+    conclusion = models.TextField(
+        blank=True,
+        help_text=_('What went well? What would change about this experience in the future?'),
```
Comment (enki211) [inline exdb/models.py]: What would you change

File: exdb/models.py [hunk 2]
```diff
@@ -59,25 +59,36 @@ class Experience(models.Model):
         ('f', _('Floor')),
     )
 
+    FUND_TYPES = (
+        ('na', _('Not necessary')),
+        ('y', _('Yes, but request not submitted yet')),
```
Comment (enki211) [inline exdb/models.py]: Maybe 'yn' instead of 'y'?

Comment (shaneallgeier) [inline exdb/models.py]: I don't care either way for this one. Anyone else feel that it should be changed?

**Comment by tom08** [comment]
Shane, if you resolve the conflicts I'll start reviewing this.

**Comment by shaneallgeier** [comment]
@tom08 Done.

**Comment by enki211** [comment]
That response time

**Comment by tom08** [comment]
Also, add the funds field to the conclusion template.

**Comment by enki211** [comment]
I approve

**Pull request merged**